src/content/developers/docs/standards/index.md

```diff
@@ -23,9 +23,20 @@ Typically standards are introduced as [Ethereum Improvement Proposals](/eips/) (
 
 ## Types of standards {#types-of-standards}
 
-Certain EIPs relate to application-level standards (e.g. a standard smart-contract format), which are introduced as [Ethereum Requests for Comment (ERC)](https://eips.ethereum.org/erc). Many ERCs are critical standards used widely across the Ethereum ecosystem.
+There are 3 types of EIPs:
 
-- [List of ERCs](https://eips.ethereum.org/erc)
+- [Standards Track](https://eips.ethereum.org/): describes any change that affects most or all Ethereum implementations
```

Comment:
I will rather call it "EIPs website".
IMHO, Pointing "Standard Track" to  https://eips.ethereum.org/ may not be appropriate.

Reply:
Yea, that's definitely a typo. Fixing